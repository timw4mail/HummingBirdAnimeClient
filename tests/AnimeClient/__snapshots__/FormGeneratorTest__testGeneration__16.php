<?php declare(strict_types=1);

return '<select id="default_list" name="default_list">
    <option value="anime">Anime</option>
    <option value="manga">Manga</option>
</select>
';
