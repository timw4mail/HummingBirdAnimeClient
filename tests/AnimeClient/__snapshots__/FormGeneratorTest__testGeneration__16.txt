<select id="default_list" name="default_list">
    <option value="anime">Anime</option>
    <option value="manga">Manga</option>
</select>
